libcst/matchers/_visitors.py

```diff
@@ -658,3 +682,24 @@ def matches(
         function.
         """
         return matches(node, matcher, metadata_resolver=self)
+
+    def findall(
+        self,
+        tree: Union[cst.MaybeSentinel, cst.RemovalSentinel, cst.CSTNode],
```

Comment:
Why support MaybeSentinel and RemovalSentinel? What's the use cases?

Reply:
As it says in the `findall` documentation: "Note that the node can also be a RemovalSentinel or a MaybeSentinel in order to use findall directly on transform results and node attributes."